specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2023-03-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,2046 @@
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "LocationInfo": {
+      "type": "object",
+      "description": "Information about location (for example: features that it supports)",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location name"
+        },
+        "capabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          },
+          "description": "List of capabilities",
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "SkuDetail": {
+      "type": "object",
+      "title": "Sku Detail",
+      "description": "Information about Sku",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The type of RedisEnterprise cluster to deploy. Possible values: (Enterprise_E10, EnterpriseFlash_F300 etc.)",
+          "enum": [
+            "Enterprise_E10",
+            "Enterprise_E20",
+            "Enterprise_E50",
+            "Enterprise_E100",
+            "EnterpriseFlash_F300",
+            "EnterpriseFlash_F700",
+            "EnterpriseFlash_F1500"
+          ],
+          "x-ms-enum": {
+            "name": "name",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Capability": {
+      "type": "object",
+      "title": "Capability",
+      "description": "Information about the features the location supports",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Feature name"
+        },
+        "value": {
+          "type": "boolean",
+          "description": "Indicates whether feature is supported or not"
+        }
+      }
+    },
+    "RegionSkuDetail": {
+      "title": "Region Sku Detail",
+      "description": "Details about the location requested and the available skus in the location",
+      "type": "object",
+      "properties": {
+        "resourceType": {
+          "type": "string",
+          "description": "Resource type which has the SKU, such as Microsoft.Cache/redisEnterprise"
+        },
+        "locationInfo": {
+          "$ref": "#/definitions/LocationInfo",
+          "description": "Details about location and its capabilities"
+        },
+        "skuDetails": {
+          "$ref": "#/definitions/SkuDetail",
+          "description": "Details about available skus"
+        }
+      }
+    },
+    "FlushParameters": {
+      "title": "Flush all the keys from the current and all its linked databases",
+      "type": "object",
+      "properties": {
+        "ids": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The resource IDs of the database resources to be flushed"
+        }
+      },
+      "required": [
+        "ids"
+      ],
+      "description": "Parameters for a Redis Enterprise Active Geo Replication FLush operation."
```

Comment:
>Redis Enterprise Active Geo Replication FLush [](http://example.com/codeflow?start=39&length=45)

capitalization etc, I'd suggest to make this 
"Parameters for a Redis Enterprise active geo-replication flush operation."

Reply:
okayy